<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border: none;
	background-color: transparent;
	background: transparent;
	padding: 0;
	margin: 0;
	font: 63 12pt &quot;Bahnschrift SemiBold Condensed&quot;;
	color: rgb(85, 255, 127);font-size: 12pt;
	color: rgb(255, 85, 0);
}
#widget{background-color: rgb(255, 255, 255,10)}
QPushButton:hover{background-color: rgb(0, 100, 255,70);}
QPushButton{
	color: rgb(0, 170, 255);
  padding: 1px 1px;
	 border-radius:7px;
	background-color: rgb(170,100, 255,50);}
#widget_2{background-color: rgb(255, 255, 255,20);}
#centralwidget{;border-left: 1px solid rgb(0, 0, 100,150)}
QSpinBox{	background-color: rgb(0, 0, 0,40);font-size:12;}
QSpinBox:hover{background-color: rgb(91, 88, 110,120);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="QSS/QSS_Resources.qrc">
              <normaloff>:/icon/Icons/window_close_focus.png</normaloff>:/icon/Icons/window_close_focus.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>33</width>
              <height>29</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>giden Veri (MB): 73.2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Gelen Veri (MB): 43.2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Harcanan  Veri (MB): 43.2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string>GB Sonra Uyar</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.150000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Başlat</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="QSS/QSS_Resources.qrc">
              <normaloff>:/icon/Icons/refresh-ccw.png</normaloff>:/icon/Icons/refresh-ccw.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>17</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(0, 255, 255);
font-size:13pt</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::TopToBottom</enum>
         </property>
         <property name="format">
          <string>%p% Günlük Uyarı limiti </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="QSS/QSS_Resources.qrc"/>
 </resources>
 <connections/>
</ui>
